From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andrew kim <andrew.kim@intel.com>
Date: Mon, 28 Jan 2019 15:35:55 -0800
Subject: [PATCH] tty: serial: fix unnecessary header file

kdb.h file it's been included in 8250_port.c file.
It doesn't provide any symbols needed.

Signed-off-by: Andrew kim <andrew.kim@intel.com>
---
 drivers/tty/serial/8250/8250_port.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/tty/serial/8250/8250_port.c b/drivers/tty/serial/8250/8250_port.c
index 775a865..75b5342 100644
--- a/drivers/tty/serial/8250/8250_port.c
+++ b/drivers/tty/serial/8250/8250_port.c
@@ -35,7 +35,6 @@
 #include <linux/nmi.h>
 #include <linux/mutex.h>
 #include <linux/slab.h>
-#include <linux/kdb.h>
 #include <linux/uaccess.h>
 #include <linux/pm_runtime.h>
 #include <linux/ktime.h>
-- 
https://clearlinux.org

